website/client/js/services/alertServices.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Alert', alertFactory);
+
+  alertFactory.$inject = [
+    '$window'
+  ];
+
+  function alertFactory($window) {
+
+    function authErrorAlert(data, status, headers, config) {
+      if (status === 0) {
+        $window.alert(window.env.t('noReachServer'));
+      } else if (status === 400 && data.errors && _.isArray(data.errors)) { // bad requests
+        data.errors.forEach(function (err) {
+          $window.alert(err.message);
```

Comment:
It could but since it's only used for authentication is should never, also it's been this way for a long time. We'll change it with the redesign